docs/source/api/arduino.rst

```diff
@@ -95,7 +95,7 @@ In this mode, we can set binary values of ``0V`` or ``5V`` to the pin.
 
 Certain sensors output analogue signals rather than digital ones, and so
 have to be read differently. The arduino has six analogue inputs, which 
-are labelled ``A0`` to ``A5``.
+are labelled ``A0`` to ``A3``.
```

Comment:
I think its misleading to say they're labelled A0 to A3, when the board clearly has six pins labelled A0 to A5. I'd much prefer we consistently say there are six pins but two of them are reserved.

```suggestion
are labelled ``A0`` to ``A5``; however pins ``A4`` and ``A5`` are reserved and cannot be used.
```